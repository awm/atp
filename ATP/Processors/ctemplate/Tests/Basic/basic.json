{
    "NAME": "Andrew",
    "ITEMS":
    [
        { "IS_LAST": false },
        { "IS_LAST": false },
        { "IS_LAST": true }
    ],
    "EMPTY_SECTION": {}
}
