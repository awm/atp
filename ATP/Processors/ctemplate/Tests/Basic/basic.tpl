Hello {{NAME}}!

This is a very basic test.

The following section does not exist:

{{#NOT_FOUND}}
I will never be printed.
{{/NOT_FOUND}}

The following section will appear three times:
{{#ITEMS}}
 * This is the last item: {{IS_LAST}}
{{/ITEMS}}

The following section will appear once:
{{#EMPTY_SECTION}}
I will be printed once.
{{/EMPTY_SECTION}}
