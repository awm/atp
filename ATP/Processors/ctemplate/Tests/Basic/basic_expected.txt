Hello Andrew!

This is a very basic test.

The following section does not exist:



The following section will appear three times:

 * This is the last item: false

 * This is the last item: false

 * This is the last item: true


The following section will appear once:

I will be printed once.

